src/System.Net.Http/src/Resources/Strings.resx

```diff
@@ -378,6 +378,9 @@
   <data name="net_http_unsupported_chunking" xml:space="preserve">
     <value>HTTP 1.0 does not support chunking.</value>
   </data>
+  <data name="net_http_unsupported_version" xml:space="preserve">
+    <value>Request specified an unsupported HTTP version.</value>
```

Comment:
Consider including in the error the requested version and also being prescriptive about what versions are allowed.